test/test_meta.py

```diff
@@ -190,6 +190,44 @@ def test_osm(self):
         out_props = self.landuse.fn(None, props, None, meta)
         self.assertEquals('scree', out_props.get('kind'))
 
+    def test_mz_is_building(self):
```

Comment:
I added tests for all the changes here to test that the yaml functions were working correctly. It already had the plumbing necessary set up to exercise these paths.